Source/PluginEditor.cpp

```diff
@@ -208,26 +210,26 @@ void SoomplerAudioProcessorEditor::mouseDrag(const MouseEvent &event)
 
 }
 
-bool SoomplerAudioProcessorEditor::isIntersectWithStartRangeLine(Point<int>* point)
+bool SoomplerAudioProcessorEditor::isIntersectWithStartRangeLine(Point<int>& point)

```

Comment:
you should declare all references as const by default and only if you need to change that argument you can remove const specifier.

Reply:
Gotcha, I will change all references to cost as I see them through the code